substrate-sgx/sp-io/with_sgx.rs

```diff
@@ -14,67 +14,63 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
```

Comment:
Is this only fmt?

Reply:
Yes, this is only formatting - apparently the file wasn't formatted before and I accidentally opened and saved it in CLion.